parley/src/context.rs

```diff
@@ -159,20 +171,36 @@ impl<'a, B: Brush, T: TextSource> RangedBuilder<'a, B, T> {
                 line_height: s.line_height,
             }
         }));
+
+        // Sort the inline boxes
```

Comment:
```suggestion
        // Sort the inline boxes so we can consume them in order
```
The old comment contains no information which isn't present in the code